resources/views/knowledgecafe/library/books/index.blade.php

```diff
@@ -13,17 +13,17 @@
     <br>
     <br>
     <div class="row">
-        <div class="col-6">
-            <h1>Books({{count($books)}})</h1>
+        <div class="col-4">
+            <h3>Books({{count($books)}})</h3    >
         </div>
         @can('library_books.create')
-        <div class="col-6">
-            <a href="{{ route('books.create') }}" class="btn btn-success float-right"><i class="fa fa-plus mr-1"></i> Add New Book</a>
+        <div class="col-8 xs-6">
+            <a href="{{ route('books.create') }}" class="btn btn-success float-right xs-2"><i class="fa fa-plus mr-1"></i> Add New Book</a>
```

Comment:
What are the means of this class `xs-2`? I think this class is not working.
The correct class is this `col-xs-2`
```suggestion
            <a href="{{ route('books.create') }}" class="btn btn-success float-right xs-2"><i class="fa fa-plus mr-1"></i> Add New Book</a>
```